example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -118,7 +118,7 @@ auto example(TAccTag const&) -> int
     using Data = std::uint32_t;
     constexpr Idx nElementsPerDim = 2;
 
-    const Vec extents(Vec::all(static_cast<Idx>(nElementsPerDim)));
+    Vec const extents(Vec::all(static_cast<Idx>(nElementsPerDim)));
```

Comment:
```suggestion
    Vec const extents(Vec::all(nElementsPerDim));
```
since `nElementsPerDim` is already of type `Idx`.